src/fio/fioEngine.js

```diff
@@ -336,7 +336,7 @@ export class FioEngine extends CurrencyEngine {
     if (!bns.eq(balance, this.walletLocalData.totalBalances[tk])) {
       this.walletLocalData.totalBalances[tk] = balance
       this.localDataDirty()
-      this.log.warn(tk + ': token Address balance: ' + balance)
```

Comment:
Contrary to our discussion, this should stay as a worn. The balance of a wallet rarely changes and it would be good to see the balance change by default in the logs